sources/packages/backend/apps/queue-consumers/src/main.ts

```diff
@@ -9,7 +9,7 @@ import { Queue } from "bull";
 import { QueueConsumersModule } from "./queue-consumers.module";
 import * as basicAuth from "express-basic-auth";
 
-async function bootstrap() {
+(async () => {
```

Comment:
just for my understanding. what was the error sonarcloud was complaining about here?

Reply:
Initially, bootstrap was called separately after its definition. Its return type was Promise<void> and therefore Sonar was complaining that the promise had to be awaited.